xcode/Subconscious/Shared/Models/Memo.swift

```diff
@@ -104,6 +104,25 @@ struct Memo: Hashable, CustomStringConvertible {
     }
 }
 
+extension Memo {
+    /// Create a draft memo with sensible default values
+    static func draft(
```

Comment:
Factor out this logic for creating draft memos.

With all the different codepaths for local vs own sphere vs other sphere, I made an effort to factor out stand-alone bits of logic like this that used to exist in `DataService`.